tools/Customization/DevHome.Customization/ViewModels/MainPageViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using Microsoft.UI.Xaml;
+using Windows.System;
+
+namespace DevHome.Customization.ViewModels;
+
+public partial class MainPageViewModel : ObservableObject
```

Comment:
Consider renaming MainPage* to CustomizationMainPage* (or something). There's also a MainPageViewModel, etc in SetupFlow, and might be useful to differentiate.

Reply:
I did consider that but then realized that the name-spacing makes that redundant so unless you have a strong preference, I think this is more concise.